### ISSUE TITLE
1.0 crashes on server

### ISSUE THREAD

**Issue by ScottKillen** [jaredlll08/Fluxed-Crystals-Reborn#19]
The Common Proxy is calling client render code. See below

```
[23:56:24] [Server thread/ERROR]: Encountered an unexpected exception cpw.mods.fml.common.LoaderException: java.lang.NoClassDefFoundError: net/minecraft/client/renderer/entity/RendererLivingEntity

    at cpw.mods.fml.common.LoadController.transition(LoadController.java:162) ~[LoadController.class:?]
    at cpw.mods.fml.common.Loader.initializeMods(Loader.java:692) ~[Loader.class:?]
    at cpw.mods.fml.server.FMLServerHandler.finishServerLoading(FMLServerHandler.java:97) ~[FMLServerHandler.class:?]
    at cpw.mods.fml.common.FMLCommonHandler.onServerStarted(FMLCommonHandler.java:319) ~[FMLCommonHandler.class:?]
    at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:210) ~[lt.class:?]
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:387) [MinecraftServer.class:?]
    at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685) [li.class:?]
Caused by: java.lang.NoClassDefFoundError: net/minecraft/client/renderer/entity/RendererLivingEntity
    at java.lang.Class.getDeclaredConstructors0(Native Method) ~[?:1.7.0_79]
    at java.lang.Class.privateGetDeclaredConstructors(Class.java:2585) ~[?:1.7.0_79]
    at java.lang.Class.getConstructor0(Class.java:2885) ~[?:1.7.0_79]
    at java.lang.Class.getConstructor(Class.java:1723) ~[?:1.7.0_79]
    at cpw.mods.fml.common.eventhandler.EventBus.register(EventBus.java:101) ~[EventBus.class:?]
    at cpw.mods.fml.common.eventhandler.EventBus.register(EventBus.java:85) ~[EventBus.class:?]
    at fluxedCrystals.proxy.CommonProxy.initialize(CommonProxy.java:25) ~[CommonProxy.class:?]
    at fluxedCrystals.FluxedCrystals.initialize(FluxedCrystals.java:94) ~[FluxedCrystals.class:?]
```

**Comment by jaredlll08** [comment]
Yea, this was fixed in 1.0.1, however I had to remove the jar because it
had other bugs

**Comment by jaredlll08** [comment]
No response and as been fixed
